spotbugs/src/main/java/edu/umd/cs/findbugs/Project.java

```diff
@@ -237,18 +237,26 @@ public boolean addFile(String fileName) {
     }
 
     /**
-     * Add a source directory to the project.
+     * Add source directories to the project.
      *
-     * @param dirName
-     *            the directory to add
-     * @return true if the source directory was added, or false if the source
-     *         directory was already present
+     * @param dirNames
+     *            the directories to add
+     * @return true if a source directory was added, or false if all source
+     *         directory were already present
      */
-    public boolean addSourceDir(String dirName) {
+    public boolean addSourceDirs(Collection<String> dirNames) {
```

Comment:
Please leave the old method there but make it deprecated. We don't want to break API yet.